Fix:
warning: extra tokens at end of #ifdef directive [-Wextra-tokens]
--- miniunz.c.orig	2010-07-18 11:04:24.000000000 -0500
+++ miniunz.c	2020-07-31 20:35:20.000000000 -0500
@@ -97,7 +97,7 @@
   SetFileTime(hFile,&ftm,&ftLastAcc,&ftm);
   CloseHandle(hFile);
 #else
-#ifdef unix || __APPLE__
+#if defined(unix) || defined(__APPLE__)
   struct utimbuf ut;
   struct tm newdate;
   newdate.tm_sec = tmu_date.tm_sec;
--- minizip.c.orig	2012-01-21 14:00:26.000000000 -0600
+++ minizip.c	2020-07-31 20:34:27.000000000 -0500
@@ -94,7 +94,7 @@
   return ret;
 }
 #else
-#ifdef unix || __APPLE__
+#if defined(unix) || defined(__APPLE__)
 uLong filetime(f, tmzip, dt)
     char *f;               /* name of file to get info on */
     tm_zip *tmzip;         /* return value: access, modific. and creation times */
